CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # RELEASE NOTES
 
+## [0.6.1] - 2020-04-10
+
+### Added
+
+- Specific error codes to various CLI operations
+
```

Comment:
Not sure if this one is worth mentioning but, https://github.com/CatalystCode/spk/pull/505 included changes that will allow new lifecycle pipelines to remove rings from the HLD once they've been removed from a project.

Another minor change is that lifecycle pipelines will only run on changes to the project `bedrock.yaml`